lib/capistrano/antistatique.rb

```diff
@@ -1,3 +1,2 @@
 require 'capistrano/antistatique/httpauth'
-require 'capistrano/antistatique/slackistrano'
 require 'capistrano/antistatique/styleguide'
```

Comment:
Maybe this file should be removed because even styleguide is optional.

Reply:
I agree. But this will be a BC break, no?

Reply:
it's a breaking change if you update the module on a project, but also an easy fix.

Reply:
And that will probably not break the build, just stop to send notifications. Also, the dependency was removed from gemspec, so makes no sense to have this here.

Reply:
OK, the `Gemfile.lock` works as expected (https://github.com/antistatique/cardis-server/blob/dev/Gemfile.lock#L2-L3)

Is the `[0.1.0](https://github.com/antistatique/cardis-server/blob/dev/Gemfile#L7)` doing something useful in the case of a reference to a git ?

Reply:
No, it's useless

Reply:
The only case that must be specified if not gem spec exist in the repository if I remember correctly